doc/radosgw/lua-scripting.rst

```diff
@@ -295,6 +300,23 @@ Operations Log
 ~~~~~~~~~~~~~~
 The ``Request.Log()`` function prints the requests into the operations log. This function has no parameters. It returns 0 for success and an error code if it fails.
 
+Background Context
+--------------------
+The background context may be used for various need such as analytics, monitoring and
+caching data from other context executions.
+
+The ``RGW`` Lua table which is accesible from any context saves the data written into it
+while execution and this data could be read and used later in any other execution.
+
+- Background script execution defualt interval is 5 seconds.
+
+- Each RGW instance has its own ``RGW`` Lua table, while the Background Context script
+  will run on every instance.
```

Comment:
I think it's important to explain to the user where this table is stored. Is it a single RADOS object? In RGW somewhere? What are the performance characteristics of serialization/de-serialization of the table?

Reply:
this is a in-memory table only, not stored in any RADOS object.
this means that the values are lost when the RGW restart, and they are not shared between RGWs.

Reply:
- Each RGW instance has its own private and ephemeral \`\`RGW\`\` Lua table that is lost when the daemon restarts.  Note that \`\`background\`\` context scripts will run on every instance.